backend/src/sms/middlewares/sms-template.middleware.ts

```diff
@@ -252,16 +253,11 @@ const pollCsvStatusHandler = async (
  */
 const deleteCsvErrorHandler = async (
   req: Request,
-  res: Response,
-  next: NextFunction
-): Promise<Response | void> => {
-  try {
-    const { campaignId } = req.params
-    await UploadService.deleteS3TempKeys(+campaignId)
-    res.sendStatus(200)
-  } catch (e) {
-    next(e)
-  }
+  res: Response
+): Promise<Response> => {
+  const { campaignId } = req.params
+  await UploadService.deleteS3TempKeys(+campaignId)
+  return res.status(200).json({ id: campaignId })
```

Comment:
i see that sometimes there's a return and sometimes there isn't. i understand it's the same but just making sure

Reply:
Because we're not catching the error here anymore (not needed) so I'm returning to keep the function signature simply to just `Response`. My convention is usually `Response` for the final handler function and `void` for middlewares in the middle